src/WorldIDIdentityManagerImplV1.sol

```diff
@@ -81,7 +82,7 @@ contract WorldIDIdentityManagerImplV1 is WorldIDImpl, IWorldID {
     SemaphoreVerifier internal semaphoreVerifier;
 
     /// @notice The interface of the bridge contract from L1 to supported target chains.
-    address internal _stateBridgeProxyAddress;
+    address internal _stateBridgeAddress;
```

Comment:
This should just be `IBridge` now. We were only taking the `address` due to it being a proxy.